### PR TITLE
Update HTS Initial form with the revised changes

### PR DESCRIPTION
**Pull request by ErickSomie**
### Description
Update the population type by adding vulnerable population
Add disability status
Update the HIV test with the 3 tier testing

### CODE REVIEW

File: configuration/ampathforms/HTS_Initial_Test.json [hunk 1]
```diff
@@ -1113,25 +1234,86 @@
...
+                    "concept": "162502AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "weeksList": ""
+                  },
+                  "id": "expiryDateB",
+                  "hide": {
+                    "hideWhenExpression": "isEmpty(testResults) || testResults !== '703AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA'"
+                  }
+                },
+                {
+                  "label": "HIV Test 2 Results:",
```
Comment (donaldkibet) [inline configuration/ampathforms/HTS_Initial_Test.json:1293]: @ErickSomie please not this should be captured as an obsGroup see [here](https://github.com/palladiumkenya/openmrs-config-kenyaemr/pull/295)

**Review by donaldkibet — CHANGES_REQUESTED** [review]

**Review by Murithijoshua — APPROVED** [review]

**Comment by makombe** [comment]
@ErickSomie Please look into the conflicts

**Comment by makombe** [comment]
Also take noted of test two result and expiry dates concepts are different from those of test one. I had aligned them #295

**Comment by donaldkibet** [comment]
@ErickSomie please resolve conflicts

**Comment by RAJABIBRAZ** [comment]
the form is outdated

**Pull request closed without merging**